rpcd/playbooks/maas-20151013-3-purge-checks.yml

```diff
@@ -0,0 +1,37 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Migrate MaaS checks and alarms to template-driven configuration
+  hosts: hosts
+  user: root
+  pre_tasks:
+    - meta: flush_handlers
+  tasks:
+    - name: Lookup Entity ID
+      shell: raxmon-entities-list | grep {{ inventory_hostname }} | awk '{gsub(/id=/,""); print $2}'
```

Comment:
Could be good to reuse the old MaaS way of doing this - I think ti would be more complete (and tested):
shell: raxmon-entities-list  | grep "label={{ physical_host|quote }}{{ maas_fqdn_extension }} " | sed -e 's/^.* id=\(.*\) label=.*$/\1/g'

More examples from: https://github.com/rcbops/rpc-openstack/blob/master/rpcd/playbooks/roles/rpc_maas/tasks/local_setup.yml